src/main.cpp

```diff
@@ -920,8 +930,9 @@ int main(int argc, char* argv[])
       break;
   }
 
-  pm.add(ast::CreateLLVMInitPass());
-  pm.add(ast::CreateClangBuildPass());
+  if (bt_debug.contains(DebugStage::Types)) {
```

Comment:
nit: I think the `CreateDumpTypesPass` could have been a separate commit but I won't block the stack.

Reply:
I wanted a way to debug it interactively. I've also added a test.